tests/common/util/TestHttpServer.cpp

```diff
@@ -134,3 +134,9 @@ TestHttpServer::handleRequest(TestHttpServer::RequestHandler handler, bool const
         boost::asio::detached
     );
 }
+
+int
+TestHttpServer::port() const
```

Comment:
You also should remove `port` parameter form constructor for `TestHttpServer` and `TestWsServer` and use `0` instead.